lib/libcouchbase/n1ql.rb

```diff
@@ -23,6 +23,14 @@ def initialize(bucket, explain: false, **options)
         attr_accessor *Ordering
         attr_accessor :explain
         attr_reader   :bucket, :connection
+        attr_accessor :string
+
+        def string(val = nil)
```

Comment:
can you explain me where do you need this method and in what it need? it's strange to return sometimes a string, sometimes an object...

Reply:
the goal is to write query like it
```bucket.n1ql.query("
          SELECT city, COUNT(*) AS cnt FROM `travel-sample`
          WHERE type = $type
          GROUP BY city
          ORDER BY cnt DESC
          LIMIT 3"```